pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -1298,6 +1298,120 @@ class Foo{
         </example>
     </rule>
 
+    <rule name="UnusedAssignment"
+          language="java"
+          since="6.26.0"
+          message="The value assigned to this variable is never used or always overwritten"
+          class="net.sourceforge.pmd.lang.java.rule.bestpractices.UnusedAssignmentRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_bestpractices.html#unusedassignment">
+        <description>
+            Reports assignments to variables that are never used before the variable is overwritten,
+            or goes out of scope. Unused assignments are those for which
+            1. The variable is never read after the assignment, or
+            2. The assigned value is always overwritten by other assignments before the next read of
+            the variable.
+
+            The rule doesn't consider assignments to fields except for those of `this` in a constructor,
+            or static fields of the current class in static initializers.
+
+            The rule may be suppressed with the standard `@SuppressWarnings("unused")` tag.
+
+            The rule subsumes UnusedLocalVariable, and UnusedFormalParameter. Those violations are filtered
```

Comment:
I'll try to make the rule names actual links. Hopefully the `{% rule` notation will work here....